### ISSUE TITLE
[macOS] since 1.10.0 direct start of jnlp via web isn't working

### ISSUE THREAD

**Issue by agido-malter** [karakun/OpenWebStart#586]
After update to 1.10.0 direct start of jnlp results in a download and exit.

macOS 14.5
CPU M2

Reproduce:
in Terminal start with `open -a 'OpenWebStart javaws.app' https://cherry.dcs.aber.ac.uk/3dexperiment/launch.jnlp`
It starts a download in my browser and exit

Old behavior in 1.9.0
Starts directly the Webstart call

All loggings are activated but no result is shown

**Comment by AlBundy33** [comment]
If you have enabled the logging in OpenWebStart Settings you'll find the logs in `~/.config/icedtea-web/log` 🤔

**Comment by agido-jrieks** [comment]
Yes, we tried that and there is nothing in there, there is not even a log file created when launched this way. The OpenWebStart logo appears shortly, then the JNLP file seems to be opened via the browser instead.
The issue occurred on several different devices.

**Comment by AlBundy33** [comment]
you're right
```
open -a "OpenWebStart javaws" https://docs.oracle.com/javase/tutorialJWS/samples/uiswing/AccessibleScrollDemoProject/AccessibleScrollDemo.jnlp
```
This opens the browser (chrome in my case) und downloads the file 🤔 

Using the same command with the local file works as expected and start the app
```
open -a "OpenWebStart javaws" AccessibleScrollDemo.jnlp
```

**Comment by valkor** [comment]
We are experiencing same issue on Mac M2.
The OpenWebStart logo appears shortly and then nothing happen - issue reported by several users after automatic upgrade.
I didn't check if jnlp downloaded somewhere, but we have to force users to downgrade to 1.9.0

**Comment by metalmolly** [comment]
We are facing the same issue on both Intel and ARM MacBooks (macOS Sonoma 14.5). No logs are persisted in `~/.config/icedtea-web/log`. Installation of 1.10.0 was done with the options 'Yes, update the existing installation' and 'For the current user only (default)'.

_**Note:** During installation of OpenWebStart 1.9.1 a file associations screen is display. This is not the case for version 1.10.0. Maybe this is related to this issue, where with 1.10.0 no file associations are created during installation?_

**Comment by arctic-itl** [comment]
We have the same problem, the OpenWebStart logo appears shortly and nothing happens. No logs are persisting in ~/.config/icedtea-web/log. Rollback to 1.9.1 resolves the problem.

**Comment by sclassen** [comment]
We will need more time to investigate this issue. Thanks for reporting

**Comment by AndreasEhret** [comment]
could you try the following: install/update to 1.10.0 and clear on ~/.cache/icedtea-web directory. Then try to restart and see if it starts again as expected. I had a similar behavior in the past (start with briefly appearing logo, nothing happens, no helpful log files) and it was because of a corrupt cache.json file in the jvm-cache directory.

**Comment by agido-malter** [comment]
> could you try the following: install/update to 1.10.0 and clear on ~/.cache/icedtea-web directory. Then try to restart and see if it starts again as expected. I had a similar behavior in the past (start with briefly appearing logo, nothing happens, no helpful log files) and it was because of a corrupt cache.json file in the jvm-cache directory.

I tried like requested, no change in behavior

**Comment by agido-malter** [comment]
> We will need more time to investigate this issue. Thanks for reporting

Could you provide a openwebstart@1.9.1 in hombrew, please? This prevents due to a brew upgrade a permanent upgrade to the broken version.

**Comment by sclassen** [comment]
Fixed with 1.10.1
